<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KernalDensity</class>
 <widget class="QDialog" name="KernalDensity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Input point or polyline features</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_LayerInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_OK">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Population field</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_Population">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Output raster</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_OutRaster">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Output cell size(optional)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_OutRasterSize">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Search radius(optional)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_SearchR">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Area units(optional)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_AreaUnit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Cancle">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_saveFilePath">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_GetInputLayer">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Values are(optional)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_outValue">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Method(optional)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_Method">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_populationField">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PB_OK</sender>
   <signal>clicked()</signal>
   <receiver>KernalDensity</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PB_Cancle</sender>
   <signal>clicked()</signal>
   <receiver>KernalDensity</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PB_GetInputLayer</sender>
   <signal>clicked()</signal>
   <receiver>KernalDensity</receiver>
   <slot>getInputLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PB_saveFilePath</sender>
   <signal>clicked()</signal>
   <receiver>KernalDensity</receiver>
   <slot>getRasterOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PB_populationField</sender>
   <signal>clicked()</signal>
   <receiver>KernalDensity</receiver>
   <slot>getPopulationField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getInputLayer()</slot>
  <slot>getRasterOut()</slot>
  <slot>getPopulationField()</slot>
 </slots>
</ui>
